integration_tests/tests/apply/about_the_person/equality_and_diversity/complete_asian_background.cy.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-param-reassign */
```

Comment:
so how does this work? You can disable a rule for a whole file but not for a block of code?

Reply:
It works just as eslint works so you can disable a block a code, but as the [package only exposes select options to be overridden](https://github.com/ministryofjustice/hmpps-typescript-lib/blob/main/packages/eslint-config-hmpps/index.cjs#L85), we can't add our own rules.

This is explained in the [readme](https://github.com/ministryofjustice/hmpps-typescript-lib/tree/main/packages/eslint-config-hmpps#customising-config). So as of right now, the only way to disable `no-param-reassign` would be to mark those files explicitly instead of a project-wide rule.